ern-orchestrator/src/runMiniApp.ts

```diff
@@ -155,6 +155,15 @@ export async function runMiniApp(
       [];
   }
 
+  if (extra?.compositeGenerator?.metroExtraNodeModules) {
```

Comment:
we can have a test in https://github.com/electrode-io/electrode-native/blob/master/ern-orchestrator/test/runMiniApp-test.ts

Reply:
Good point, I added a test :+1: